zilliqa/src/transaction.rs

```diff
@@ -736,6 +820,40 @@ pub struct TransactionReceipt {
     pub exceptions: Vec<ScillaException>,
 }
 
+impl TransactionReceipt {
+    pub fn hash(&self) -> Hash {
+        let success = [u8::from(self.success); 1];
+        let accepted = [u8::from(self.accepted.unwrap_or_default()); 1];
+        Hash::compute([
```

Comment:
What's the spec for this? Raided from ZQ1?

Reply:
@JamesHinshelwood @DrZoltanFazekas  ? Want to make a decision and we can put it in the docs? If you give a transaction a hash, someday someone is going to want to verify it ..